CBAC.json

```diff
@@ -1837,13 +2081,1011 @@
                 }
             ],
             "versions": "EQ(*)"
+        },
+        {
+            "action": "get file instance",
+            "description": "Searches for file instances",
```

Comment:
Does this action search for a single file instance or multiple? the `action` and `description` make it confusing

Reply:
Action can be return multiple instances, hence change the action name to `get file instances`